playbooks/aws/openshift-cluster/install.yml

```diff
@@ -21,5 +21,45 @@
 - name: run the std_include
   include: ../../common/openshift-cluster/std_include.yml
 
-- name: run the config
-  include: ../../common/openshift-cluster/config.yml
+- name: Verify Requirements
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+
+  - name: Verify Requirements - EL
+    when: ansible_distribution != "Fedora"
+    action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - package_version
+      - docker_image_availability
+      - docker_storage
+  - name: Verify Requirements - Fedora
+    when: ansible_distribution == "Fedora"
+    action: openshift_health_check
+    args:
+      checks:
+      - docker_image_availability
+
+- include: ../../common/openshift-etcd/config.yml
+
+- include: ../../common/openshift-nfs/config.yml
+  when: groups.oo_nfs_to_config | default([]) | count > 0
+
+- include: ../../common/openshift-loadbalancer/config.yml
+  when: groups.oo_lb_to_config | default([]) | count > 0
+
+- include: ../../common/openshift-master/config.yml
+
+- include: ../../common/openshift-master/additional_config.yml
+
+- include: ../../common/openshift-node/config.yml
```

Comment:
Is this line intended?

Reply:
@michaelgugino,

The masters run as nodes.  This is for those hosts and not for the bootstrapped groups.  We can probably consider doing all of them as bootstrapped.

Reply:
Oh yeah, I always forget masters are also nodes.

Reply:
Yeah, in GCP everything is bootstrapped which caused #6011 to fail when we enabled those by default